tests/test262-esnext-excludelist.xml

```diff
@@ -8062,4 +8062,6 @@
   <test id="built-ins/Atomics/waitAsync/bigint/null-bufferdata-throws.js"><reason></reason></test>
   <test id="built-ins/Atomics/waitAsync/null-bufferdata-throws.js"><reason></reason></test>
   <!-- END - Missing test262 support in JerryScript REPL - missing $262.detachArrayBuffer function -->
+  <test id="built-ins/ArrayBuffer/prototype/slice/species-returns-not-arraybuffer.js"><reason></reason></test>
+  <test id="built-ins/SharedArrayBuffer/prototype/slice/species-returns-not-arraybuffer.js"><reason></reason></test>
```

Comment:
I think these tests reveal some error in this PR. Please check them, I dont think they are need to be added to skiplist.